Source/Wpf/Prism.Wpf/Common/WindowStartupLocationBehavior.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Windows;
+
+namespace Prism.Common
+{
+    public static class WindowStartupLocationBehavior
```

Comment:
This does not follow my recommended pattern for declaring an attached property. Please follow the pattern set by the DialogWindowStyleProperty in the DialogService.cs.  I am also hesitant to add this because this address only one property. What if you need to modify the value of other properties that are not DPs?  This is why I didn't add this yet, as this would become a pain if we had to add an attached property for each requested CLR property of the Window class.  I was thinking of some type of property bag approach to make it more generic.  This is also something someone could add to their projects without having to add it directly to Prism source if required.

Reply:
If this class was added, it would not go in common because nothing else uses it, only dialogs, so it should be in the dialogs namespace

Reply:
> What if you need to modify the value of other properties that are not DPs?
Correct me if i am wrong, but i think that the only CLR property which does not have a dp, is  WindowStartupLocation. A more complex solution will be an overkill maybe?
Also you are right that someone could add this to his project if needs to, but i suppose that the majority of the people, that will use the IDialogService, will need it at least 1 time. That's why prism can provide it as a part of it's idialogservice solution.